server/services/nomisPushService.js

```diff
@@ -15,15 +16,31 @@ module.exports = (nomisClientBuilder, signInService) => {
     const nomisClient = nomisClientBuilder(systemTokens.token)
 
     logger.info('Pushing approval status to nomis', approvalStatus)
-    return nomisClient.putApprovalStatus(bookingId, approvalStatus)
+    try {
+      return await nomisClient.putApprovalStatus(bookingId, approvalStatus)
+    } catch (error) {
+      if (error.status === 409) {
+        logger.error(`409 CONFLICT when posting approval status for booking id: ${bookingId}`)
+        throw nomisPushError('Approval Status')
+      }
+      throw error
+    }
   }
 
   async function pushChecksPassed({ bookingId, passed, username }) {
     const systemTokens = await signInService.getClientCredentialsTokens(username)
     const nomisClient = nomisClientBuilder(systemTokens.token)
 
     logger.info(`Pushing checks passed=${passed} to nomis for bookingId: ${bookingId}`)
-    return nomisClient.putChecksPassed({ bookingId, passed })
+    try {
+      return await nomisClient.putChecksPassed({ bookingId, passed })
```

Comment:
the is no point doing a `return await` you can just do 
```
return nomisClient.putChecksPassed({ bookingId, passed })
``` 
since the caller does the waiting..

But since you are handling the errors in this function rather then the caller I guess it's not a problem 👍

Reply:
yep, have to do it to catch the errors. Just return means the errors don't go in to the catch